playwright-e2e/page-objects/citizen/payments/ctzSubmittedPage.page.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page } from '@playwright/test';
+
+export class CTZSubmittedPage {
```

Comment:
CTZ could be confusing?

Reply:
Could be shortened to CIT for Citizen? What are your thoughts on this?

Reply:
Does it have to be shortened?  If yes, CIT is definitely clearer.

Reply:
It makes sense to shorten it as file name could become larger. I'll rename the convention to CIT for ctizen